_posts/2018-01-02-project-open-qa.markdown

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+img: openqa.png
+title: openQA
+type: project
+technologies: [Perl]
+website: http://open.qa/
+github: https://github.com/os-autoinst/openQA
```

Comment:
Actually you can leave it just at the organizational level if you want

Reply:
Can I change or should I change it? 😉

Reply:
It's completely up to you :) Project is openQA but the org is os-autoinst 📄